src/utils/proposals.ts

```diff
@@ -0,0 +1,40 @@
+import { DataViewerProposal } from '../interfaces/DataViewerProposal';
+
+const PROPOSAL_APPLICANT_NAME_CASCADE = [
+  'organization_name',
+  'organization_dba_name',
+  'organization_legal_name',
+  'proposal_primary_contact_name',
+  'proposal_submitter_name',
+];
+
+const PROPOSAL_APPLICANT_NAME_FALLBACK = 'Unknown Applicant';
+
+/**
+   * Proposals may store the applicant name in one of several fields.
+   * This utility returns iterates through the list of attributes where
+   * we expect to find a name, in a preferential order, and returns the
+   * first matching value.
+   *
+   * If none is found, falls back to a fallback value (if configured) or null.
+   *
+   * @param  {DataViewerProposal} proposal
+   * @param  {Boolean} fallback Whether to fall back to the applicant name fallback
+   *                            if no better name is found. Defaults to true.
```

Comment:
Under what circumstances would we set this to `false`?

Reply:
😆 An excellent question. I don't have a current need. The circumstances I imagined were checking "does this proposal have an applicant name?" (separate from "give me an applicant name, even if it's the fallback string"). But if you'd prefer I remove it until we need it, happy to.

Reply:
Yes, please! :upside_down_face:

Reply:
Done!